apps/greenhouse/package.json

```diff
@@ -15,105 +15,44 @@
   "private": true,
   "devDependencies": {
     "@cloudoperators/juno-config": "*",
-    "@svgr/core": "^7.0.0",
     "@svgr/plugin-jsx": "^7.0.0",
     "@tailwindui/react": "^0.1.1",
-    "@testing-library/dom": "^8.19.0",
     "@testing-library/jest-dom": "^5.17.0",
```

Comment:
do we still need `@testing-library/jest-dom`. This one is already removed from the other apps.